<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBDMXLedSettingsUi</class>
 <widget class="QWidget" name="OpenRGBDMXLedSettingsUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>43</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QSpinBox" name="RedSpinBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="LedLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="GreenSpinBox"/>
   </item>
   <item row="0" column="3">
    <widget class="QSpinBox" name="BlueSpinBox"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
